docs/core/building/index.md

```diff
@@ -0,0 +1,107 @@
+---
+title: Build .NET Core from source
+description: Build .NET Core from source
+keywords: .NET, .NET Core, source, build
+author: beleroy
+ms.author: mairaw
+ms.date: 03/29/2017
+ms.topic: article
+ms.prod: .net-core
+ms.devlang: dotnet
+ms.assetid: 8b49079c-6ede-429a-92d7-ecd2fda1ab0e
+---
+
+# Build .NET Core from source
+
+The ability for anyone to build .NET Core from its source code is important in multiple ways: it makes it easier to port .NET Core to new platforms, it enables contributions and fixes to the product, and it enables the creation of custom versions of .NET.
+This section of the documentation gives guidance to developers who want to build and distribute their own versions of .NET Core.
+
+## How to build from source
+
+### Building the CLR
+
+The source code for the .NET Core CLR can be found in [the `dotnet/coreclr` repository on GitHub](https://github.com/dotnet/coreclr/).
+
+The build currently depends on Git, CMake, Python and a C++ compiler.
+Once these prerequisites are installed, the CLR is built by invoking the 'build' script (`build.cmd` or `build.sh`) at the base of [the Core CLR repository](https://github.com/dotnet/coreclr/).
+
+The details of installing the components differ depending on the operating system.  See the following pages based on your OS.
+There is no cross-building across OS (only for ARM, which is built on X64).  
+You have to be on the particular platform to build that platform.  
+
+ * [Windows Build Instructions](https://github.com/dotnet/coreclr/blob/master/Documentation/building/windows-instructions.md)
```

Comment:
if you change the introduction to say they're build instructions, then you don't need to say in each link